package.json

```diff
@@ -12,6 +12,10 @@
   "bin": {
     "run": "bin/run.js"
   },
+  "scripts": {
+    "build": "babel src/ --out-dir lib/",
+    "test": "mocha"
```

Comment:
That's true, but this gives a more consistent development story. If I'm building I issue `npm run build`. If I'm testing, I issue `npm test`. If run is installed then we could just have run map back to these npm commands. It's a bit convoluted, but the thing you want to avoid is the build breaking if run is broken. If you're using run to build run but the last built version of run was broken then you can't build a new version of run without doing the build manually. I ran into that several times during development. This way we can run `build` and `test` without explicitly relying on run.